src/e2e/java/teammates/e2e/cases/StudentProfilePageE2ETest.java

```diff
@@ -42,6 +42,27 @@ public void testAll() {
         profilePage.editProfilePhoto();
         profilePage.verifyPhotoSize("220px", "220px");
 
+        ______TS("Typical case: Profile picture ratios");
+        profilePage.fillProfilePic("src/test/resources/images/profile_pic_too_wide.jpg");
+        profilePage.uploadPicture();
+        profilePage.verifyStatusMessage("Your profile picture has been saved successfully");
+
+        profilePage.showPictureEditor(); // In order to upload
+        profilePage.waitForUploadEditModalVisible();
+
+        profilePage.editProfilePhoto();
+        profilePage.verifyPhotoMaxHeight(220, "src/test/resources/images/profile_pic_too_wide.jpg");
```

Comment:
I don't find this necessary. The test pictures used are already known beforehand and thus the dimensions can be readily determined. Just use the same method `verifyPhotoSize` with the correct values.